Update reporting.md (#1143)
telerik/blazor-docs: integrations/reporting.md

# Reporting

You can utilize the [Telerik Reporting](https://www.telerik.com/products/reporting) product to embed reporting functionality in your Blazor app. We provide the following components for easy integration (find detailed information in the respective articles that are part of the Telerik Reporting product documentation).

* [Telerik Blazor Report Viewer](#telerik-blazor-report-viewer)
* [Telerik Native Blazor Report Viewer](#telerik-native-blazor-report-viewer)
* [Telerik Blazor Report Designer Component](#telerik-blazor-report-designer)

## Telerik Blazor Report Viewer

> The [Telerik Blazor Report Viewer Component](https://docs.telerik.com/reporting/blazor-report-viewer) enables you to embed a report preview into you app. The component is built on top of our robust and reliable HTML5 Report Viewer - a JavaScript/jQuery widget with lots of customization options. See this report viewer component into action in the [Dashboard Report Demo](https://demos.telerik.com/reporting/dashboard). This report viewer includes the following functionality:

Reviewer: ```suggestion
The [Telerik Blazor Report Viewer Component](https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/blazor-report-viewer/overview) enables you to embed a report preview into you app. The component is built on top of our robust and reliable HTML5 Report Viewer - a JavaScript/jQuery widget with lots of customization options. See this report viewer component into action in the [Dashboard Report Demo](https://demos.telerik.com/reporting/dashboard). This report viewer includes the following functionality:
```

* Preview the pages of the resulting report document on screen;
* Print the report;
* Export the report into various formats (e.g., PDF, Word, Excel);
* Search in the report content;
* Use any other interactive features that the Telerik Reporting product offers.

## Telerik Native Blazor Report Viewer

The [Telerik Native Blazor Report Viewer Component](https://docs.telerik.com/reporting/embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/overview) is a report viewer built with **native Blazor components** from our Telerik UI for Blazor library (requires a valid license). This report viewer brings a fluent user experience that blends with the rest of your Blazor application using the same input controls and styling mechanism. The initial version of this viewer supports all the core functionality needed for previewing and exporting report documents.

## Telerik Blazor Report Designer

> The [Telerik Blazor Report Designer Component](https://docs.telerik.com/reporting/how-to-blazor-web-report-designer) brings embedding of the report designer tool into your Blazor application. This empowers your trusted end-users to edit existing or create new report definitions from scratch. See the report designer into action in the [Web Report Designer Demo](https://demos.telerik.com/reporting/designer)

Reviewer: ```suggestion
The [Telerik Blazor Report Designer Component](https://docs.telerik.com/reporting/how-to-blazor-web-report-designer) enables you to embed a Report Designer tool in your Blazor application. This empowers your trusted end-users to edit existing or create new report definitions from scratch. See the Report Designer into action in the [Web Report Designer Demo](https://demos.telerik.com/reporting/designer)
```
